src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -704,179 +705,135 @@ export default class PhotoField extends React.Component {
+                <ProgressBar percent={this.state.progress}/>
+              </div>}
+              {fieldView === 'cropper' && <div className="cropper-container-outer">
+                <Cropper
+                  ref="cropper"
+                  key={smallScreen ? 'smallScreen' : 'largeScreen'}
+                  ready={this.setCropBox}
+                  responsive
+                  src={this.state.src}
+                  aspectRatio={1}
+                  cropBoxMovable={false}
+                  cropstart={this.handleCropstart}
+                  cropend={this.handleCropend}
+                  cropmove={this.handleCropMove}
+                  minContainerHeight={smallScreen ? SMALL_CROP_BOX_SIZE : LARGE_CROP_BOX_SIZE}
+                  toggleDragModeOnDblclick={false}
+                  dragMode="move"
+                  guides={false}
+                  viewMode={0}
+                  zoom={this.onZoom}/>
+                <div className="cropper-zoom-container">
+                  {smallScreen && <button className="cropper-control cropper-control-zoom cropper-control-zoom-out va-button va-button-link" type="button" onClick={() => this.handleZoomButtonClick('OUT')}><i className="fa fa-search-minus"></i></button>}
+                  {!smallScreen && <button className="cropper-control cropper-control-zoom cropper-control-zoom-out va-button va-button-link" type="button" onClick={() => this.handleZoomButtonClick('OUT')}>
+                    <span className="cropper-control-label">Make smaller<i className="fa fa-search-minus"></i></span>
+                  </button>}
+                  <input type="range"
+                    ref="slider"
+                    className="cropper-zoom-slider"
+                    min={this.state.minRatio}
+                    max={this.state.maxRatio}
+                    defaultValue="0.4"
+                    step="0.01"
+                    aria-valuemin={this.state.minRatio}
+                    aria-valuemax={this.state.maxRatio}
+                    aria-valuenow={this.state.zoomValue}
+                    onMouseDown={this.onZoomMouseDown}
+                    onMouseUp={this.onZoomMouseUp}
+                    onMouseMove={this.onZoomMouseMove}
+                    onChange={this.onZoomSliderChange}/>
+                  {smallScreen && <button className="cropper-control cropper-control-zoom cropper-control-zoom-in va-button va-button-link" type="button" onClick={() => this.handleZoomButtonClick('IN')}><i className="fa fa-search-plus"></i></button>}
+                  {!smallScreen && <button className="cropper-control cropper-control-zoom cropper-control-zoom-in va-button va-button-link" type="button" onClick={() => this.handleZoomButtonClick('IN')}>
+                    <span className="cropper-control-label">Make larger<i className="fa fa-search-plus"></i></span>
+                  </button>}
+                </div>
+                <CropperControls
```

Comment:
new component

Reply:
The name of this sort of implies to me that the range slider would be in it as well.

Reply:
That's true- I'll rework the markup / classNames a bit to make that work.